packages/edit-site/src/components/dataviews/list-view.js

```diff
@@ -57,10 +68,34 @@ function HeaderMenu( { dataView, header } ) {
 		>
 			{ isSortable && (
 				<DropdownMenuGroupV2>
-					<FieldSortingItems
-						field={ header.column }
-						dataView={ dataView }
-					/>
+					{ Object.entries( sortingItemsInfo ).map(
+						( [ direction, info ] ) => (
+							<DropdownMenuItemV2
+								key={ direction }
+								prefix={ <Icon icon={ info.icon } /> }
+								suffix={
+									sortedDirection === direction && (
+										<Icon icon={ check } />
+									)
+								}
+								onSelect={ ( event ) => {
+									event.preventDefault();
+									if ( sortedDirection === direction ) {
+										dataView.resetSorting();
```

Comment:
I see you separated the code again here. That's fine, but the plan is to update the code here to update the view, right?

Reply:
I don't really know whether we should merge the two, especially since list view heavily depends on tanstack. We'll see how that menu grows.